ui/library/src/components/DownloadButton.tsx

```diff
@@ -35,9 +39,9 @@ export const DownloadButton: React.FunctionComponent<Props> = ({
       disabled={fetching}
       onClick={() => download()}
       aria-label="Download PDF"
-      className="reader__download-btn"
+      className={classname ? `reader__download-btn ${classname}` : 'reader__download-btn'}
```

Comment:
- Are you planning on adding a custom class from the S2 side? Just curious what this would be used for that couldn't be accomplished by targeting the `reader__download-btn` class.
- This can be simplified to ``className={`reader__download-btn ${className}`}`` and `classname` should have a capital N

Reply:
@carolinepaulic that's precisely what im trying to do. Using the classname that similar to other button will be more consistent for our code. Also isnt `className` is a react thing

Reply:
Alright, I like the idea of consistent classNames. `className` is a react thing, but isn't that what you're trying to mimic? It looks like we use `className` for these props in S2 as well, for example in the `CLIconButton`:
![image](https://user-images.githubusercontent.com/7637347/161130337-5f4f7e0a-3906-4d28-b864-6118c83133c2.png)

My main concern is that it might not be obvious to someone that they need to use `classname` here when `className` is the standard.